### ISSUE TITLE
Session\Adapter\Database never starts

### ISSUE THREAD

**Issue by yergo** [phalcon/incubator#318]
I found that after changes of e3b10837444f166fc56e1c7e1685f35ecb72ad45, where method `::start()` was overriden, sessions running on database adapter are never starting - at least not where their are configured in way described on readme file.

No errors, just not working. Calling `start()` method does nothing too. Looks like `parent::start()` is required somewhere.

Edit:

Got a quick fix for it:

    public function start()
    {
        if (!$this->isStarted) {
            $this->isStarted = parent::start();
        }

        return $this->isStarted;
    }

but I don't know this code enough to dare in commiting it. Also object has declared:

```
  protected 'isStarted' => boolean true
  protected '_started' => boolean true
```
to there is a bit too much anyway.

**Comment by sergeyklay** [comment]
@yergo Can u do PR please?

**Comment by yergo** [comment]
Not really. First of all, I'm writing this from job and this issue already stopped me for a while. Second thing is I would gladly revert whole e3b10837444f166fc56e1c7e1685f35ecb72ad45, but I have no idea if I should because of 

> overridden, otherwise the start method will register a PHP session which
will not/should not be used.

Feeling like something is behind this.

**Comment by sergeyklay** [comment]
@Mechzeit ?

**Comment by sergeyklay** [comment]
Solved via #319
